modules/acquia_cms_common/config/optional/filter.format.filtered_html.yml

```diff
@@ -20,17 +20,16 @@ filters:
     settings: {  }
   filter_url:
     id: filter_url
-    provider: filterclear
+    provider: filter
```

Comment:
@panshulK - I think updating provider is the only key required here, can we keep other changes intact if possible. Do let me know if I misunderstood.